l10n_ch_fds_import_bank_statement/models/fds_postfinance_file.py

```diff
@@ -55,7 +55,14 @@ def import_to_bank_statements(self):
         except UserError as e:
             # wrong parser used, raise the error to the parent so it's not
             # catch by the following except Exception
-            raise e
+            if e.name == _('This file doesn\'t contain any statement.'):
```

Comment:
The best would be to create a new class that inherits UserError and check here if the error is an instance of this class, instead of checking the message (with translation that could introduce some problems?)

Reply:
Done, I was not sure about where to place the new error class and used a translated string to compare but it was a pretty objectionable decision. Thank you for the feedback.